features/old/applications/providers/create_application.feature

```diff
@@ -78,6 +78,7 @@ Feature: Create application
    Scenario: The service of the selected application plans hasn´t service plan
      Given a service "second" of provider "foo.example.com"
        And a published application plan "second_app_plan" of service "second" of provider "foo.example.com"
+       And the service "second" does not have service plan
```

Comment:
![image](https://user-images.githubusercontent.com/64276/64427921-0dc53300-d0bb-11e9-8485-81e996babd54.png)

A default service plan is created everytime a Service is created. But in case it is deleted, there was a feature that shows a link to create a service plan. Though this link leads to nowhere

So we either delete this entire test and the feature, or fix the broken link or the broken behaviour (in case the service plan is deleted)